python/triton/language/core.py

```diff
@@ -1179,7 +1179,7 @@ def arange(start, end, _builder=None):
     """
     Returns contiguous values within the half-open interval :code:`[start,
     end)`.  :code:`end - start` must be less than or equal to
-    :code:`TRITON_MAX_TENSOR_NUMEL = 131072`
+    :code:`TRITON_MAX_TENSOR_NUMEL = 1048576`
```

Comment:
we should use directly `TRITON_MAX_TENSOR_NUMEL` to avoid it from getting out of date

Reply:
Updated